spring-cloud-deployer-kubernetes/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -250,10 +251,7 @@ PodSpec createPodSpec(AppDeploymentRequest appDeploymentRequest) {
 		podSpec.withTolerations(this.deploymentPropertiesResolver.getTolerations(deploymentProperties));
 
 		// only add volumes with corresponding volume mounts
```

Comment:
This comment needs to be moved to the new location of where we add volumes to volume mounts.